lib/LaTeXML/Package.pm

```diff
@@ -164,6 +164,8 @@ sub coerceCS {
   if    (ref $cs) { }
   elsif ($cs =~ s/^\\csname\s+(.*)\\endcsname//) {
     $cs = T_CS('\\' . $1); }
+  elsif ($cs =~ s/^(.)$//) {    # Match an active char
+    ($cs) = TokenizeInternal($1)->unlist; }
```

Comment:
[minor, optional] I would have probably checked `length($cs) == 1` to avoid the regex hoop, but it works either way.

Reply:
OK, thanks.